pkg/controller/persistentvolume/persistentvolume_controller.go

```diff
@@ -484,7 +482,164 @@ func (ctrl *PersistentVolumeController) syncBoundClaim(claim *api.PersistentVolu
 func (ctrl *PersistentVolumeController) syncVolume(volume *api.PersistentVolume) error {
 	glog.V(4).Infof("synchronizing PersistentVolume[%s]: %s", volume.Name, getVolumeStatusForLogging(volume))
 
-	return nil
+	// [Unit test set 4]
```

Comment:
Having these comments sprinkled around is great, but I am very worried it will drift.  Can we make teh linkage more durably?